catalogue/webapp/pages/concept.tsx

```diff
@@ -335,6 +340,44 @@ export const ConceptPage: NextPage<Props> = ({
   );
 };
 
+function getDisplayIdentifierType(identifierType: IdentifierType): string {
+  switch (identifierType.id) {
+    case 'lc-names':
+      return 'LC Names';
+    case 'lc-subjects':
+      return 'LCSH';
+    case 'nlm-mesh':
+      return 'MeSH';
```

Comment:
Yes, lowercase 'e' is the correct capitalisation. https://www.nlm.nih.gov/mesh/meshhome.html